README.md

```diff
@@ -1,29 +1,106 @@
-    - Export Frames Until (How far to export)
-    - Labels (Labels for tagging)
- 5. Tag the video frame by frame.
- 6. Export Video to CNTK Format
- 7. Train model using [Object Detection using FastRCNN] (https://github.com/Microsoft/CNTK/wiki/Object-Detection-using-Fast-R-CNN#train-on-your-own-data).
- 8. Load new video apply model to new video, validate tags, re-export, retrain
- 9. Repeat step #8 on new videos until model preformance is satisfactory.  
-
-## Upcoming Features
+# CNTK Object Detection Video Tagging Tool
+
+This tool provides end to end support for generating datasets for and validating Object Detection Models with CNTK.
+
+**It supports the following scenarios:**
+
+- Computer assisted tagging and tracking of objects in video using the [Camshift tracking algorithm](http://opencv.jp/opencv-1.0.0_org/docs/papers/camshift.pdf) 
+- Exporting tags and assets to CNTK format for training a CNTK object detection model.
+- Running and validating a trained CNTK object detection model on new videos to generate stronger models. 
+
+## Table of Contents
+ - [Prerequisites](#prerequisites)
+ - [Tagging Job](#taggingjob)
+ - [Review/Iterate CNTK Object Detection Model](#review)
+ - [Upcoming Features](#upcoming)
+ - [How to Contribute](#contribute)
+ 
+-----------
+<a name="prerequisites"></a>
+## Prerequisites
+**Please Note that installation of CNTK and the FAST-RCNN dependencies is only required for CNTK model review and training, not tagging.**
+
+ 1. Install [CNTK](https://github.com/Microsoft/CNTK/wiki/Setup-CNTK-on-your-machine) (*Note: currently the tool only supports the non-pip/full installation of CNTK.*)
+ 2. Follow the setup instruction of the [CNTK Fast-RCNN tutorial](https://github.com/Microsoft/CNTK/wiki/Object-Detection-using-Fast-R-CNN#setup) (*Note: Fast-RCNN  is only supported for Linux with python 3.4 and not 3.5*)
+ 3. Download and Extract the app [release package](https://github.com/CatalystCode/CNTK-Object-Detection-Video-Tagging-Tool/releases)
+ 4. Configure CNTK-Config.json with the following properties
+ 
+  ```json
+   {
+    "cntkPath" : "{CNTK Path default is c:/local/cntk}", 
+   }
+  ```
+ 5. Run the app
+ 
+-----------
+<a name="taggingjob"></a>
+## Tagging Job 
+ 1. Load a mp4 video file by either dragging it into the app or clicking and selecting it.
+  
+    <img src="/media/2_load.jpg" alt="Home Page" height="426" width="695"/>
+   
+ 2. Configure the tagging job specifying the following preferences
+ 
+    <img src="/media/3_Job_Configuration.jpg" alt="Home Page" height="586" width="685"/>
+ 
+    **Frame Extraction Rate:** The number of frames to tag per a second of video.<br>
+    
+    **Tagging Region Type:**  The type of bounding box for tagging regions.<br>
+      - "Rectangle": Tag bounding boxes of any dimension.
+      - "Square": Tag bounding boxes of auto-fixed dimensions.
+      - "Point": Tag a point (This is useful for object recognition but not object detection)
```

Comment:
Remove "Point" (For now just remove it from the dropdown selection, no need to change other code)